src/main/java/org/apache/log4j/jdbc/JDBCAppender.java

```diff
@@ -17,382 +17,127 @@
 package org.apache.log4j.jdbc;
 
 import java.sql.Connection;
-import java.sql.DriverManager;
 import java.sql.SQLException;
-import java.sql.Statement;
 import java.util.ArrayList;
-import java.util.Iterator;
 
-import org.apache.log4j.PatternLayout;
-import org.apache.log4j.spi.ErrorCode;
+import org.apache.log4j.helpers.LogLog;
 import org.apache.log4j.spi.LoggingEvent;
 
 
 /**
-  The JDBCAppender provides for sending log events to a database.
-  
- <p><b><font color="#FF2222">WARNING: This version of JDBCAppender
- is very likely to be completely replaced in the future. Moreoever,
- it does not log exceptions</font></b>.
-
-  <p>Each append call adds to an <code>ArrayList</code> buffer.  When
-  the buffer is filled each log event is placed in a sql statement
-  (configurable) and executed.
-
-  <b>BufferSize</b>, <b>db URL</b>, <b>User</b>, & <b>Password</b> are
-  configurable options in the standard log4j ways.
-
-  <p>The <code>setSql(String sql)</code> sets the SQL statement to be
-  used for logging -- this statement is sent to a
-  <code>PatternLayout</code> (either created automaticly by the
-  appender or added by the user).  Therefore by default all the
-  conversion patterns in <code>PatternLayout</code> can be used
-  inside of the statement.  (see the test cases for examples)
-
-  <p>Overriding the {@link #getLogStatement} method allows more
-  explicit control of the statement used for logging.
-
-  <p>For use as a base class:
-
-    <ul>
-
-    <li>Override <code>getConnection()</code> to pass any connection
-    you want.  Typically this is used to enable application wide
-    connection pooling.
-
-     <li>Override <code>closeConnection(Connection con)</code> -- if
-     you override getConnection make sure to implement
-     <code>closeConnection</code> to handle the connection you
-     generated.  Typically this would return the connection to the
-     pool it came from.
+  The JDBCAppender provides for sending log events to a database
+  in Log4j up to 1.2.17.
 
-     <li>Override <code>getLogStatement(LoggingEvent event)</code> to
-     produce specialized or dynamic statements. The default uses the
-     sql option value.
-
-    </ul>
-
-    @author Kevin Steppe (<A HREF="mailto:ksteppe@pacbell.net">ksteppe@pacbell.net</A>)
+  Changed in 1.2.18+ to complain about its use and do nothing else.
+  See <a href="https://logging.apache.org/log4j/1.2/">the log4j 1.2 homepage</a>
+  for more information on why this class is disabled since 1.2.18.
 
+  @author Kevin Steppe (<A HREF="mailto:ksteppe@pacbell.net">ksteppe@pacbell.net</A>)
```

Comment:
We no longer use author tags in general. Credit is attributed in pom.xml and/or changes.xml.

Reply:
I didn't add any author tags, but also didn't remove any. Actual change here is only consistent whitespace indenting, Github diff is not displaying it too cleanly.